public/sections/todo/todo.html

```diff
@@ -4,10 +4,12 @@
         <input type="text" placeholder="Search" ng-model="findTodo">
     </li>
     <li class="collection-item" ng-repeat="todo in ctrl.todos | filter : findTodo">
-        <a><i class="material-icons waves-circle tiny " style="background-color : {{todo.color}}">circle</i></a>
+        <a><i class="material-icons waves-circle tiny padding-circle" style="background-color : black; ">circle</i></a>
```

Comment:
This should remain as `{{ todo.color }}` If it wasn't showing for you, it's because the default color for todos is white, so they're not visible. I'll change that to our ambient color blue-gray.